### ISSUE TITLE
Remove dependency on MSYS or Cygwin

### ISSUE THREAD

**Issue by mikedougherty** [ifwe/digsby#8]
Requiring two shell replacements on windows makes me sad :(

**Comment by letmaik** [comment]
Why is this needed anyway? Similarly, all of those native dependencies, are there really no pure-python alternatives at least for some of them? Now and then I have the thought of working a bit on Digsby's code but everytime I see those massive build instructions, dependencies, build scripts etc., I'm just not motivated to even start. In my opinion, the reason why there is no development happening is not because Digsby is written in Python, but because of this initial entry barrier.

So, in theory, how simple could Digsby's dev experience be made? Where are the limits? Which complex build steps / tools / native libraries are absolutely needed?

**Comment by haveric** [comment]
You're exactly right. The build process needs a huge cleanup. I've attempted the whole build process about 10-15 times and each time I got stuck at a different place. My last attempt, I finally got all of the sanity checks to pass and just ran into more issues: https://github.com/ifwe/digsby/issues/36

My hope was that once I got it working once, I could set it up on a VM to get other developers quicker access until some of the build steps were improved.

The other issue at this point is that most of the services (at least that I used) have either discontinued their apis or switched to OAuth2, which will require re-implementation.

I really don't want this project to fade out of existence, but I've sunk enough hours into it getting nowhere that I feel defeated. I can't imagine many other developers even getting as far as I have with this.

**Comment by Brok3nHalo** [comment]
Yeah, the build process is definitely the biggest hurdle. I've tried several times to get it up running without much luck. I think I somehow got it to build once in a VM, but with all sorts of missing dependancies. If I remember correctly I think a majority of the prereqs is because it's using a non-standard build of wxWidgets and/or wxWebKit, if it was modified to be able to use vanilla wxPython and webkit I think it would eliminate a significant number of pre-reqs.

Once past that I think updating the various protocols wont be too bad, most the broken ones I think are just minor repointing calls to the correct api version and not major logic changes.
